examples/load-manifests-examples.js

```diff
@@ -9,7 +9,9 @@ function addExamples () {
   const examples = [
     '-',
     'http://demo.unified-streaming.com/video/ateam/ateam.ism/ateam.mpd',
-    'http://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd'
+    'http://storage.googleapis.com/shaka-demo-assets/angel-one/dash.mpd',
+    'https://dash.edgesuite.net/akamai/bbb_30fps/bbb_30fps.mpd',
```

Comment:
I think in repo there should be only stable examples and if served via https the certificate shouldn't be self signed. I think we can create a separate file, something like "manifests_examples.md" where we can add more edge cases with some explanations what it does and how and when to use it.

Reply:
I do not agree. The sample is the only way you can check there is no regression in the library. And each time a pb appears on a stream, we must add it to the sample, so that, on next delivery, we'll be able to test it without having to modify the sample code.

Moreover, how could you test the corretion of the pr if the stream is not in the sample ?